di-demo/server/node/index.js

```diff
@@ -76,7 +78,7 @@ const init = async () => {
       app.use(
         "/",
         proxy({
-          target: "http://app:3000",
+          target: "http://fe:9000",
```

Comment:
Kijk ik eroverheen of mis ik de aangepaste docker-compose file waarin `links: - fe:app` is vervangen voor `links: - fe:fe`?

Reply:
scherp! die is niet nodig dus blijkbaar